src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php

```diff
@@ -636,4 +650,47 @@ private function assertVersionStrategy(ContainerBuilder $container, Reference $r
             $this->assertEquals($format, $versionStrategy->getArgument(1));
         }
     }
+
+    private function assertCachePoolServiceDefinitionIsCreated(ContainerBuilder $container, $name, $type, $defaultLifetime, $namespaceArgumentIndex = null)
+    {
+        $id = 'cache.pool.'.$name;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid variables with short names like $id. Configured minimum length is 3.](https://www.codacy.com/app/git_9/symfony/file/2331802253/issues/source?bid=3207821&fileBranchId=3207834#l656)